pages/common/spike.md

```diff
@@ -0,0 +1,19 @@
+# spike
+
+> Spike is a full featured static website generator written in JavaScript.
+
+- Create a new project using the default template:
+
+`spike new {{project_name}}`
+
+- Compile your project, watche for changes, and reload:
```

Comment:
`watche` -> `watching`. Typo?

Also, reload what?

Reply:
Reload the website in case of any file changes. Perhaps we should also mention that it opens the site in a browser.